apps/src/applab/applab.js

```diff
@@ -250,6 +250,11 @@ function adjustAppSizeStyles(container) {
               childRules[k].style.cssText = "right: " +
                   Applab.appWidth * scale + "px;";
               changedChildRules++;
+            } else if (childRules[k].selectorText === "#visualizationResizeIcon") {
```

Comment:
also, the `changedChildRules < 8` above should now be `changedChildRules < 9`. Feel free to move the number to a constant :smile: